client/extensions/zoninator/components/forms/zone-content-form/post-placeholder.jsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+
+const PostPlaceholder = () => {
+	const className = 'zoninator__zone-list-item is-placeholder';
```

Comment:
If you named this class `zone-content-form__placeholder` then you wouldn't have any linter issues and you could get rid of the `className` variable, which would make this component even more succinct. To be absolutely certain of no name collisions, the CSS selector would then be `.zoninator__main .zone-content-form__placeholder`.